src/main/java/time/TimeInCapitals/TimeInCapitalsApplication.java

```diff
@@ -2,12 +2,20 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import time.TimeInCapitals.config.TimeInCapitalsConfig;
+import time.TimeInCapitals.data.CapitalsData;
 
 @SpringBootApplication
 public class TimeInCapitalsApplication {
 
 	public static void main(String[] args) {
 		SpringApplication.run(TimeInCapitalsApplication.class, args);
+		
+		AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(TimeInCapitalsConfig.class);
+		
+		CapitalsData timeInCapitalsConfig = (CapitalsData) context.getBean("getCapitalsData");
```

Comment:
Beany można wyciągać w ten sposób, natomiast nie jest to konieczne w naszym przypadku.

Reply:
Podpowiesz coś więcej? :) Chodzi o inny sposób wyciągania w tej klasie beana?

Reply:
Nie potrzebujesz tutaj nic wyciągać z kontekstu, te dwie linie są do wyrzucenia.